app/views/shared/_navigation.html.haml

```diff
@@ -12,7 +12,7 @@
       %li{ class: navigation_item_class(organisation_activities_path(organisation_id: current_user.organisation_id)) }
         = link_to t("page_title.activity.index"), organisation_activities_path(organisation_id: current_user.organisation_id), class: "govuk-header__link"
 
-      - if policy(:export).index?
+      - if policy([:export, Organisation]).index?
```

Comment:
(Just making notes commit by commit, ignore if not relevant by the final commit!)

Should we take the opportunity to check the policy against an actual organisation rather than the generic permission?

Reply:
Nah, the index policy is only for listing all organisations, so doesn't have an organisation to check against.